model/task/task_test.go

```diff
@@ -3140,6 +3141,124 @@ func TestArchiveMany(t *testing.T) {
 	}
 }
 
+func TestArchiveManyFailedOnly(t *testing.T) {
+	require.NoError(t, db.ClearCollections(Collection, OldCollection))
+	t1_et1 := Task{
```

Comment:
nit: can these be re-written to use camelCase instead of snake_case? (just et1 / et2 would be reasonable)